package.json

```diff
@@ -60,12 +61,14 @@
   "devDependencies": {
     "@openmrs/esm-framework": "next",
     "@openmrs/esm-patient-common-lib": "next",
+    "@playwright/test": "^1.52.0",
     "@swc/core": "^1.7.14",
     "@swc/jest": "^0.2.36",
     "@testing-library/dom": "^8.20.1",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.5.2",
+    "@types/dotenv": "^8.2.3",
```

Comment:
I don't think we need this anymore. It's way behind dotenv, which I think now has it's own typings.